lib/order.rb

```diff
@@ -0,0 +1,25 @@
+class Order
+
+  def initialize
+    @list = {}
+  end
+
+  def add(dish, quantity)
+    @list.store(dish.to_sym, quantity)
```

Comment:
what happens if they then add another of the same dish?
```ruby
order.add(:fish, 2)
order.add(:fish, 1)
```
I would expect to see 3 fish now.